### PR TITLE
Minor cleanup of windows build script and venv configuration

### PR DESCRIPTION
**Pull request by dnwillia-ansys**

### CODE REVIEW

**Comment by akaszynski** [comment]
This is helpful. Add the usage of the `configure_venv.bat` to `README.rst`.

**Review by MaxJPRey — APPROVED** [review]

**Comment by akaszynski** [comment]
Add a windows batch file file and this is good to go.

File: configure_venv.bat [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@echo off
+setlocal
```
Comment (MaxJPRey) [inline configure_venv.bat:2]: @dnwillia-ansys Out of curiosity, why do we need to use `setlocal` here as we don't define any variable?

Comment (dnwillia-ansys) [reply configure_venv.bat:2]: Yeah, it's technically not needed here.  Force of habit from my side.

Comment (MaxJPRey) [reply configure_venv.bat:2]: That's fine by me. Thanks Dan.

**Pull request merged**